koku/masu/database/sql/openshift/ui_summary/reporting_ocp_gpu_summary_p.sql

```diff
@@ -0,0 +1,66 @@
+DELETE FROM {{schema | sqlsafe}}.reporting_ocp_gpu_summary_p
+WHERE usage_start >= {{start_date}}::date
+    AND usage_start <= {{end_date}}::date
+    AND source_uuid = {{source_uuid}}
+;
+
+INSERT INTO {{schema | sqlsafe}}.reporting_ocp_gpu_summary_p (
+    id,
+    cluster_id,
+    cluster_alias,
+    namespace,
+    node,
+    pod,
+    usage_start,
+    usage_end,
+    gpu_vendor_name,
+    gpu_model_name,
+    gpu_memory_capacity_mib,
+    gpu_hours,
+    gpu_count,
+    gpu_cost,
+    unallocated_gpu_cost,
+    source_uuid,
+    cost_category_id,
+    raw_currency,
+    cost_model_cpu_cost,
+    cost_model_memory_cost,
+    cost_model_volume_cost,
+    cost_model_rate_type
+)
+    SELECT uuid_generate_v4() as id,
+        cluster_id,
+        cluster_alias,
+        namespace,
+        node,
+        max(all_labels->>'pod-name') as pod,
+        usage_start,
+        usage_start as usage_end,
+        max(all_labels->>'gpu-vendor') as gpu_vendor_name,
+        max(all_labels->>'gpu-model') as gpu_model_name,
+        max((all_labels->>'gpu-memory-mib')::numeric) as gpu_memory_capacity_mib,
+        sum((all_labels->>'gpu-uptime-hours')::numeric) as gpu_hours,
+        count(DISTINCT resource_id) as gpu_count,
+        sum(cost_model_gpu_cost) as gpu_cost,
+        NULL as unallocated_gpu_cost,
+        source_uuid,
+        cost_category_id,
+        max(raw_currency) as raw_currency,
+        sum(cost_model_cpu_cost) as cost_model_cpu_cost,
+        sum(cost_model_memory_cost) as cost_model_memory_cost,
+        sum(cost_model_volume_cost) as cost_model_volume_cost,
+        max(cost_model_rate_type) as cost_model_rate_type
+    FROM {{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary
+    WHERE data_source = 'GPU'
+        AND usage_start >= {{start_date}}::date
+        AND usage_start <= {{end_date}}::date
+        AND source_uuid = {{source_uuid}}
+    GROUP BY cluster_id,
+        cluster_alias,
+        namespace,
```

Comment:
For the other places where we have separate UI summary tables for project & node, it is because those columns are not in the base table. 
1. [Network](https://github.com/project-koku/koku/blob/main/koku/masu/database/sql/openshift/ui_summary/reporting_ocp_network_summary_p.sql)
2. [Cost](https://github.com/project-koku/koku/blob/main/koku/masu/database/sql/openshift/ui_summary/reporting_ocp_cost_summary_p.sql)

We are including namespace & node in the base table here, so I am not really seeing the need for two other tables to just group by project & node.